app/build.gradle

```diff
@@ -60,41 +60,44 @@ android {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
+
+    // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.5.0'
-    implementation 'androidx.appcompat:appcompat:1.3.0'
+
+    // AndroidX
     implementation 'androidx.activity:activity-ktx:1.2.3'
-    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1'
-    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
-    implementation 'androidx.fragment:fragment-ktx:1.3.4'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    implementation 'androidx.preference:preference-ktx:1.1.1'
-    implementation 'androidx.recyclerview:recyclerview:1.2.0'
+    implementation 'androidx.appcompat:appcompat:1.3.0'
     implementation 'androidx.cardview:cardview:1.0.0'
-    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
-    implementation 'androidx.vectordrawable:vectordrawable:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'androidx.core:core-ktx:1.5.0'
     implementation 'androidx.documentfile:documentfile:1.0.1'
+    implementation 'androidx.fragment:fragment-ktx:1.3.4'
     implementation 'androidx.hilt:hilt-work:1.0.0'
+    kapt 'androidx.hilt:hilt-compiler:1.0.0'
```

Comment:
Nice to see that some kind of structuring has been defined for the dependencies. However, seeing the `kapt` entries mixed with the `implementation` entries seems weird (just a personal preference). Do you think you could move all `kapt` entries to a separate "block", kind of like they were previously?

Also, it looks like you have organized dependencies alphabetically. However, the "Third-party" dependencies don't follow this ordering completely. Would be nice to keep this consistency, that is, moving the "com." packages above the "io." packages.

Reply:
Ah, I only put the AndroidX libraries in alphabetical order. I'll put them in complete alphabetical order then.